style.scss

```diff
@@ -0,0 +1,98 @@
+body, h1 {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    background-color: rgb(226, 226, 226);
+}
+
+.container {
+    max-width: 25em;
+    margin: 1em auto;
+    background-color: white;
+    box-shadow: 0 0 10px rgb(172, 172, 172);
+}
+
+.todo {
+    margin-top: .5em;
+    padding-top: .5em;
+}
+
+.todo__content {
+    padding: 1em;
+}
+
+.todo__header {
+    text-align: center;
+}
+
+/* New Todo Entry Field */
+.todo__new-todo-field {
+    display: block;
+    box-sizing: border-box;
+    width: 100%;
+    font-size: 1em;
+    padding: .5em;
+}
+
+.todo__complete-all-field {
+    
+}
+.checkbox-field--small {
+    font-size: .75em;
+}
+
+.todo-items {
+    margin: 1em 0;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+.todo-item {
+    font-size: 1.5em;
+    padding: .5em 0;
+    border-bottom: 1px rgba(128, 128, 128, 0.342) solid;
+    position: relative;
```

Comment:
You may want to consider trying flexbox here instead of positioning some of the items absolute. It gives you the ability to easily vertically center too!

Reply:
Yup, will work on flexbox on its own feature branch now